readme.txt

```diff
@@ -35,6 +35,12 @@ We're always interested in your feedback and our [premium forums](https://theeve
 
 == Changelog ==
 
+= 2.0.0 2020-10-26 =
+
+* Extension now works with both Legacy (V1) and updated (V2) calendar designs
```

Comment:
add `Fix - `, `Tweak - `, or `Feature - ` prefix to each.
add period at the end of each.

Reply:
Done.